doc/user-guide/Push-notifications.md

```diff
@@ -31,29 +31,31 @@ This module communicates with XMPP clients directly in order to enable/disable n
 
 The [mod_push][] module is very easy to enable - just paste the following to your MongooseIM configuration file:
 
- ```erlang
+```
 {mod_push, [
     {wpool, [{workers, 100}]}
 ]}.
- ```
+```
 
 And that's basically it.
 You have just enabled the push notification support with 100 asynchronous workers that will handle all push notification related work.
 
-Let's now consider the case when you need to control the format of push notifications or the state of the user sessions more robustly.
-In that case you need to create a plugin module that implements the `mod_push_plugin` behaviour and enable this plugin as specified in [mod_push][] documentation.
+You can also control the format of "sender" of push notification (which ultimately becomes the title of push
```

Comment:
You can also control the format of the "sender" of the push notification (which ultimately becomes the title of the push notification) and filter which messages will trigger the notification.